server/app/controllers/NewTask.scala

```diff
@@ -19,6 +19,7 @@ class NewTask @Inject()(repo: Repo, app: play.api.Application, val messagesApi:
   val addTaskForm = Form {
     mapping(
       taskDescription -> nonEmptyText,
+      solutionTrait -> nonEmptyText,
```

Comment:
we better discuss this task :-) Need to redone.
solutionTrait should be taken on fly by server code via regexp. Just reuse this code from Runner:
val solutionTrait = traitDefPattern.findFirstIn(suite) match {
      case Some(v) => v.split("""\s+""")(1)
      case None => throw new RuntimeException(s"There is no trait type defined in the Test constructor, code: $suite")
    }